angler/config/system-proprietary-blobs-api24.txt

```diff
@@ -1,5 +1,6 @@
 system/bin/fingerprintd
 system/bin/mm-qcamera-app
+system/bin/nanoapp_cmd
```

Comment:
The only difference between production build and AOSP generated build is some meaningless compiler metadata stored under `note.gnu.build-id` section. Functionality is identical thus no reason to include at the blobs configuration files.

```
$ diff <(xxd aosp_prod/out_release_Linux_7.0.0_r4/aosp_prod/target/product/bullhead/system/bin/nanoapp_cmd) <(xxd nexus-vendor-blobs/bullhead/nrd90s/factory_imgs_data/system/bin/nanoapp_cmd)
40,41c40,41
< 00000270: 0300 0000 474e 5500 9853 2424 7b3d e593  ....GNU..S$${=..
< 00000280: 0bbe 7a17 7c2b 91b2 0000 0000 0000 0000  ..z.|+..........
---
> 00000270: 0300 0000 474e 5500 b9a3 83ca 002c e2b1  ....GNU......,..
> 00000280: 10ac 85a4 4659 e523 0000 0000 0000 0000  ....FY.#........
$ objdump -h aosp_prod/out_release_Linux_7.0.0_r4/aosp_prod/target/product/bullhead/system/bin/nanoapp_cmd | grep .note.gnu.build-id
  2 .note.gnu.build-id 00000020  0000000000000268  0000000000000268  00000268  2**2
```